README.md

```diff
@@ -28,6 +28,15 @@ Questions, bug reporting and issue tracking are provided by GitHub. Please
 report all bugs by creating a new issue with the bug tag. You can ask
 questions by creating a new issue with the question tag.
 
+Contributing
+------------
+
+Contributing to DTK is easy. Just send us a [pull request](https://help.github.com/articles/using-pull-requests/).
```

Comment:
This can be more heavily documented on readthedocs.  Anyway, "send us a pull request with your contributions and associated unit tests" sounds good to me.  Let's not waste more time on this.